agent/exploits/cve_2024_11639.py

```diff
@@ -12,22 +10,21 @@
 from agent import exploits_registry
 from agent import definitions
 
-
-VULNERABILITY_TITLE = "Path Traversal in Ivanti CSA"
-VULNERABILITY_REFERENCE = "CVE-2024-8963"
-VULNERABILITY_DESCRIPTION = """Path Traversal in the Ivanti CSA before 4.6 Patch 519 allows a remote unauthenticated attacker to access restricted functionality."""
+VULNERABILITY_TITLE = "Authentication Bypass in Ivanti CSA Admin Console"
+VULNERABILITY_REFERENCE = "CVE-2024-11639"
+VULNERABILITY_DESCRIPTION = """An authentication bypass in the admin web console of Ivanti CSA before 5.0.3 allows a remote unauthenticated attacker to gain administrative access."""
 RISK_RATING = "CRITICAL"
 
-"""Ivanti has released a security advisory for CVE-2024-8963, a critical vulnerability in Ivanti CSA 4.6 which was 
-incidentally addressed in its patch for CVE-2024-8190, which was released on September 10, 2024 (CSA 4.6 Patch 519)."""
-PATCH_DATE = datetime.datetime(2024, 9, 10)
+PATCH_DATE = datetime.datetime(
+    2024, 12, 10, 17, 58, 17
+)  # Based on Last Modified Date in advisory
```

Comment:
I'm not sure I follow, if the description says `before 5.0.3`, why is the check done on patch date & not by version?

Reply:
Fingerprinting the version wasn't feasible, so I worked with the patch date instead.